sycl/doc/EnvironmentVariables.md

```diff
@@ -28,6 +29,73 @@ compiler and runtime.
 
 `(*) Note: Any means this environment variable is effective when set to any non-null value.`
 
+### `ONEAPI_DEVICE_SELECTOR`
+
+With no environment variables set to say otherwise, all platforms and devices presently on the machine are available. The default choice will be one of these devices, usually preferring a LevelZero GPU device, if available. The `ONEAPI_DEVICE_SELECTOR` can be used to limit that choice of devices, and to expose GPU sub-devices (aka "tiles") as individual devices.  
```

Comment:
```suggestion
With no environment variables set to say otherwise, all platforms and devices presently on the machine are available. The default choice will be one of these devices, usually preferring a LevelZero GPU device, if available. The `ONEAPI_DEVICE_SELECTOR` can be used to limit that choice of devices, and to expose GPU sub-devices or sub-sub-devices as individual devices.  
```

I recently learned that the sub-devices are not necessarily tiles, so we should remove all the wording which equates "sub-device"  with "tile".  We should just call them sub-devices.

Search for "tile" in the wording below and make similar changes.